schemas/case-counts/admin0/v1/admin0.json

```diff
@@ -0,0 +1,293 @@
+{
+  "name": "Admin0 Case Counts",
+  "description": "",
+  "filename": {
+    "regex": "^admin0\\.csv$"
```

Comment:
@hafen Do we want to keep these filename restrictions (in each schema file)? Maybe just change them all to `^*\\.csv$`?

Reply:
Yes I think we can make it more generic, just .csv.